### ISSUE TITLE
INC, INX, INY

### ISSUE THREAD

**Issue by AlexCouch** [AlexCouch/6502-cr#6]
Increment instructions, one for an address, one for X, and one for Y

```
inc #$3f47    ;Increment the value at the given address
inx                 ;Increment the X register
iny                 ;Increment the Y register
```
- [x] INC
- [x] INX
- [x] INY

**Comment by AlexCouch** [comment]
INC implemented in this commit: 7c80785950de5742909c7e0f91cf493fa3c8d6a6

**Comment by AlexCouch** [comment]
INX, INY implemented in this commit: 5e25b92b16e11563d236d8d197d51d66eedd7e74
